### ISSUE TITLE
I am wondering about the detailed hyperparameters for summarization with dynamic convolutions (Rouge-1=39.84).

### ISSUE THREAD

**Issue by hong8e** [pytorch/fairseq#576]
I am wondering about the detailed hyperparameters for summarization with dynamic convolutions (Rouge-1=39.84).

fp16 training = ?
encoder, decoder kernel size = 3,7,15,31*4 ?
one batch = ? tokens
gradient_accum= ? batches
warmup steps = ?
lr warmup steps from  = 10^-7 to 10^-3 ?
total training steps = ?

thank you!

**Comment by huihuifan** [comment]
yes fp16, encoder: 3,7,15,31x4, decoder: 3,7,15,31x3, max-tokens was set to 3584, gradient accumulation is 16 batches, 4000 warmup updates, from 1e-7 to 1e-3
